ruby/ql/lib/change-notes/2024-02-12-raw-erb-output.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Raw output ERb tags of the form `<%== ... %>` are now recognised as cross-site scripting sinks.
```

Comment:
`ERb` looks a little strange, should it be `Erb` or `ERB` ?

Reply:
Hmm yes, I thought `ERb` was the correct casing (for "Embedded Ruby") but after some research it looks like I just invented that. Everyone else uses ERB.